etc/env/deploy.env

```
@@ -0,0 +1,16 @@
+# In Mac / Windows replace the 172.17.0.1 with host.docker.internal
+DATABASE_URL=postgres://postgres@172.17.0.1/plasma
+FORCED_EXIT_REQUESTS_WEB3_URL=http://172.17.0.1:4444
```

```suggestion
DATABASE_URL=postgres://$DB_USER:$DB_PASSWORD@$DB_HOST:5432/$DB_NAME
FORCED_EXIT_REQUESTS_WEB3_URL=http://localhost:4444
```
Can we have this instead? I'll have a script in the server that updates this file with the variables.

---

you can create your own file and just update the `ENV_OVERRIDE` variable.

---

Also, the localhost in `FORCED_EXIT_REQUESTS_WEB3_URL` can be variable too.

---

We already discussed and address it